client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -57,216 +54,186 @@ const StockListComponent: FC = () => {
+        defaultHideOnMobile: true,
+        enableSorting: true,
+      },
+      {
+        header: t('label.pack-quantity'),
+        accessorKey: 'totalNumberOfPacks',
+        columnType: ColumnType.Number,
+        align: 'right',
+        size: 125,
+        enableSorting: true,
+      },
+      {
+        header: t('label.soh'),
+        description: t('description.soh'),
+        accessorFn: row => row.totalNumberOfPacks * row.packSize,
+        align: 'right',
+        enableSorting: false,
         defaultHideOnMobile: true,
-        Cell: UnitsAndMaybeDosesCell,
+        Cell: UnitsAndDosesCell,
       },
+      {
+        id: 'availableStockOnHand',
+        header: t('label.available-soh'),
+        description: t('description.available-soh'),
+        accessorFn: row => row.availableNumberOfPacks * row.packSize,
+        align: 'right',
+        enableSorting: false,
+        defaultHideOnMobile: true,
+        Cell: UnitsAndDosesCell,
+      },
+      {
+        accessorKey: 'costPricePerPack',
+        header: t('label.pack-cost-price'),
+        description: t('description.pack-cost'),
+        columnType: ColumnType.Currency,
+        size: 125,
+        defaultHideOnMobile: true,
+        enableSorting: true,
+      },
+      {
+        id: 'totalCost',
+        header: t('label.total'),
+        accessorFn: row => row.totalNumberOfPacks * row.costPricePerPack,
+        columnType: ColumnType.Currency,
+        description: t('description.total-cost'),
+        enableSorting: false,
+        size: 125,
+        defaultHideOnMobile: true,
+      },
+      {
+        id: 'supplierName',
+        header: t('label.supplier'),
+        accessorFn: row =>
+          row.supplierName ? row.supplierName : t('message.no-supplier'),
+        Cell: TextWithTooltipCell,
+        size: 190,
+        defaultHideOnMobile: true,
+        enableSorting: true,
+      },
+      ...(plugins.stockLine?.tableColumn || []),
     ],
-    {
-      key: 'costPricePerPack',
-      label: 'label.pack-cost-price',
-      description: 'description.pack-cost',
-      Cell: CurrencyCell,
-      width: 125,
-      defaultHideOnMobile: true,
-    },
-    {
-      key: 'totalValue',
-      label: 'label.total',
-      accessor: ({ rowData }) =>
-        rowData.totalNumberOfPacks * rowData.costPricePerPack,
-      Cell: CurrencyCell,
-      description: 'description.total-cost',
-      sortable: false,
-      width: 125,
-      defaultHideOnMobile: true,
-    },
-    {
-      key: 'supplierName',
-      label: 'label.supplier',
-      accessor: ({ rowData }) =>
-        rowData.supplierName ? rowData.supplierName : t('message.no-supplier'),
-      Cell: TooltipTextCell,
-      width: 190,
-      defaultHideOnMobile: true,
-    }
-    // TODO: re-enable when this table is updated, plugin column API is for the new table implementation
-    // ...(plugins.stockLine?.tableColumn || [])
+    [t, manageVvmStatusForStock, plugins.stockLine?.tableColumn]
```

Comment:
Roxy was saying we don't normally bother adding `t` to the deps array, seems to be the case. Shouldn't cause problems as it's instantiated way upstream and will never change.